interfaces/Portalicious/src/app/pages/auth-callback/auth-callback.page.html

```diff
@@ -0,0 +1 @@
+<ng-container *ngComponentOutlet="CallbackComponent"></ng-container>
```

Comment:
Should we maybe add `app-page-layout` here? To minimize page jumping around (especially if we use this also for basic-auth - see the other comment about this).